packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -0,0 +1,167 @@
+# Accordion
+
+An accordion is used to allow users to toggle between different content sections in a compact vertical stack.
+
+This document outlines the API of `Accordion` component.
+
+<div align="center">
+  <img src="./accordion-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Accordion](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-619941&t=Rcrabprdzrz7JdId-0)
+
+## Interactions
+
+### Accordion open
+
+https://github.com/razorpay/blade/assets/6682655/1a02d680-1832-4a70-a0a3-1a1f56f2b8b9
+
+### Accordion close
+
+https://github.com/razorpay/blade/assets/6682655/b54eaeb9-7257-4da4-bf3b-89e10785e4f9
+
+## API
+
+Annotated component with props ([figma](https://www.figma.com/file/wzCTQ9kMjWWHKFu7kCzYGi/Accordion-breakdown?type=design&node-id=0-1&t=GfrkTz89y2M46wQN-0)):
```

Comment:
Need access

<img width="549" alt="Screenshot 2023-05-30 at 10 36 28 PM" src="https://github.com/razorpay/blade/assets/29686866/1bb4f70b-a01c-4af3-a841-1163f5d4479a">

Reply:
Found the design [here][1], I can access this one. Please update the link.

[1]: https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=31097-721626&t=S5YhzX8uYWUwLZqj-4

Reply:
That one has restricted access. It's an annotated version of the component with props (screenshot included in the markdown). The full component figma is the one prior to this section.

Reply:
Why add restricted links in a public doc even Razorpay employees don't have access to? 😕

Reply:
Just kept it for record, will clean up if it's causing inconvenience :)